config/locales/user_mailer/fr.yml

```diff
@@ -179,11 +179,11 @@ fr:
       subject: Les résultats de la vérification en personne de %{app_name} sont reportés
     in_person_please_call:
       body:
-        contact_message_html: Appelez le <strong>%{contact_number}</strong> et indiquez
-          le code d’erreur <strong>%{support_code}</strong>.
+        contact_message_html: Appelez le <strong>%{contact_number}</strong> pour leur
+          communiquer le code d’erreur <strong>%{support_code}</strong>.
         intro_html: Appelez notre centre de contact avant le <strong>%{date}</strong>
```

Comment:
I am a little confused on the translations. Sorry if I am wrong but maybe double check this. Should the intro_html be `d’ici le` rather than `avant le`. I am looking the top of page 3 [here](https://docs.google.com/document/d/1xi_djTWB-iWgXjVQtBUgcyRAU6b9Qpsa8K-LEh2lJCg/edit). 

Like so... Appelez notre centre de contact `d’ici le` 15 avril 2023 pour continuer la vérification de votre identité.

Also, I initially thought translations were coming in for image alt text (see red [here)](https://docs.google.com/document/d/1ssNa_Eq36chOkQCkkE0lTgZelMg5Sux4czQ6rzRASeI/edit)

Reply:
@allis-green  Ahh, yes. I was confused. 😵‍💫  I had the wrong line. I will resolve this comment. Sorry about that.